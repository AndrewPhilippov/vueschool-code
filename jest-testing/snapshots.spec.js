const user = {
    name: 'Andrew Philippov',
    age: 33,
    job: 'Frontend Engineer'
}

test('User matches', () => {
    expect(user).toMatchSnapshot()
})
