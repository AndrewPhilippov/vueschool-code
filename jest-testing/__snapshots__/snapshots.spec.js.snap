// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User matches 1`] = `
Object {
  "age": 33,
  "job": "Frontend Engineer",
  "name": "Andrew Philippov",
}
`;
